lib/grpc/error.ex

```diff
@@ -8,18 +8,24 @@ defmodule GRPC.RPCError do
 
       # client side
       {:error, error} = Your.Stub.unary_call(channel, request)
+
+  See GRPC.Status for more details on possible statuses.
```

Comment:
```suggestion
  See `GRPC.Status` for more details on possible statuses.
```

This will add the link automatically

Reply:
Fixed [f48e9d6](https://github.com/elixir-grpc/grpc/pull/314/commits/f48e9d6677db4caa481119b67ec5ec4aecd5d917)